packages/drivers/odsp-driver/src/test/odspError.spec.ts

```diff
@@ -133,7 +157,10 @@ describe("Odsp Error", () => {
 			},
 		};
 		const networkError = errorObjectFromSocketError(socketError, "error");
-		if (networkError.errorType !== DriverErrorType.genericNetworkError) {
+		if (
+			networkError.errorType !== OdspErrorTypes.genericNetworkError ||
```

Comment:
This shouldn't be necessary. The `isIGenericNetworkError` check should be sufficient.
```suggestion
```